narayana-release-process.sh

```diff
@@ -150,32 +150,32 @@ fi
 mvn clean install -DskipTests -gs tools/maven/conf/settings.xml -Dorson.jar.location=$ORSON_PATH -Pcommunity
 if [[ $? != 0 ]]
 then
-  echo Could not install narayana
+  echo 1>&2 Could not install narayana
```

Comment:
Hi Ondra, I will marked as approved as the main changes look good. I am not sure of the need for the 1>&2 on the echo, can you explain please?

Reply:
It's used in the script (e.g. https://github.com/jbosstm/narayana/blob/master/narayana-release-process.sh#L55) so I think that was some decision to write the errors to the `STD_ERR` as defaul way. If that's not the good idea for these places I can remove it and let it being written to `STD_OUT`.

Reply:
I think it is fine - thanks for the explanation!